framework/assets/yii.js

```diff
@@ -219,9 +216,10 @@ window.yii = (function ($) {
                 }
                 if (!/(get|post)/i.test(method)) {
                     $form.append($('<input/>', {name: '_method', value: method, type: 'hidden'}));
-                    method = 'POST';
+                    method = 'post';
+                    $form.attr('method', method);
                 }
-                if (!/(get|head|options)/i.test(method)) {
+                if (/post/i.test(method)) {
```

Comment:
Having check against GET, HEAD and OPTIONS methods is redundant here, because `method` will be only `get` or `post` ath this point (in case of other methods, for example HEAD variable `method` will be assigned to `post` in previous check).